c45bot.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Maybe remove the `.sh` from the script. After that I'll merge as I'm happy with the rest.